File: C_Programs/InputNUmbersAndPrintArray4x5.cpp
//create a 2 d array of 4 rows and 5 column

#include<stdio.h>
#include<conio.h>

int main()
{
	int num[4][5];
	int i,j;
	for(i=0;i<4;i++)
	{
		for(j=0;j<5;j++)
		{
			printf("\nEnter any number : ");
			scanf("%d",&num[i][j]);
		}
	}
	for(i=0;i<4;i++)
	{
		for(j=0;j<5;j++)
		{
			printf("%d\t",num[i][j]);
		}
		printf("\n");
	}
	
	return 0;
}

File: Basic_Java_Programs/InputNumbersAndPrintArray4x5.java
//create a 2 d array of 4 rows and 5 column
import java.util.*;

public class InputNumbersAndPrintArray4x5 {
    public static void main(String[] args)
    {
        int[][] num=new int[4][5];
        int i,j;
        Scanner sc=new Scanner(System.in);

        for(i=0;i<4;i++)
        {
            for(j=0;j<5;j++)
            {
                System.out.println("Enter any number : ");
                num[i][j]=sc.nextInt();
            }
        }
        for(i=0;i<4;i++)
        {
            for(j=0;j<5;j++)
            {
                System.out.print(num[i][j]+"\t");
            }
            System.out.println();
        }
    }
}
